pinecone/sparse/splade/splade-vector-generation.ipynb

```diff
@@ -1220,9 +1230,10 @@
       ]
     },
     {
-      "attachments": {},
       "cell_type": "markdown",
-      "metadata": {},
+      "metadata": {
+        "id": "lXDl8lvoGz9P"
+      },
```

Comment:
Again, it seems weird to end the notebook with a DF, then link to the next notebook.

If this is the notebook that agedly generated that dataset, you need to show the upload command that uploads this notebook to `https://storage.googleapis.com/pinecone-datasets/`.  
It's ok if this command will fail for users since they don't have write permissions.  We should explain that this is how we generated the Quora dataset which is used throughout our Pincecone example notebooks, and that users are welcome to generated their own datasets by replacing the destination address to an S3 \ GCS bucket that they own.

Reply:
@jamescalam WDYT?

Reply:
I think it's a broader thing we need to consider so I prefer to not hold the PR for this right now